ManageWikiNamespaces.php

```diff
@@ -141,4 +141,14 @@
 		],
 		'requires' => [],
 	],
+	'wgARENamespaces' => [
+		'name' => 'Enable Article Ratings in this namespace',
+		'from' => 'articleratings',
+		'type' => 'check',
+		'main' => true,
+		'talk' => false,
+		'blacklisted' => [],
+		'overridedefault' => false,
```

Comment:
Should be default to be on on $wgContentNamespaces (https://github.com/wikimedia/mediawiki-extensions-ArticleRatings/blob/5ca30436ade43d27edc9cb2669d2ab8395d06734/ArticleRatingsHooks.php#L10)